exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -272,60 +285,54 @@ func (b *BTCMarkets) FetchOrderbook(p currency.Pair, assetType asset.Item) (orde
 // UpdateOrderbook updates and returns the orderbook for a currency pair
 func (b *BTCMarkets) UpdateOrderbook(p currency.Pair, assetType asset.Item) (orderbook.Base, error) {
 	var orderBook orderbook.Base
-	orderbookNew, err := b.GetOrderbook(p.Base.String(),
-		p.Quote.String())
+	strPair := b.FormatExchangeCurrency(p, assetType).String()
+	tempResp, err := b.GetOrderbook(strPair, 2)
```

Comment:
Can just call b.FormatExchangeCurrency directly without creating strPair as you're not using it again